ddtrace/sampling_rule.py

```diff
@@ -171,4 +179,9 @@ def __eq__(self, other):
         if not isinstance(other, SamplingRule):
             raise TypeError("Cannot compare SamplingRule to {}".format(type(other)))
 
-        return self.sample_rate == other.sample_rate and self.service == other.service and self.name == other.name
+        return (
```

Comment:
Where are we comparing samplingrules?

Reply:
I don't know of any code that uses this, but it's part of the public API.